### ISSUE TITLE
2.0 /cluster/xxx/metrics/namespace/org.graylog2 - returns ApiError: Unexpected character

### ISSUE THREAD

**Issue by SjonHortensius** [Graylog2/graylog2-server#1771]
I've configured graylog as described in #1766 and almost everything works fine, but I'm having strange issues with one of the requests:

`$ curl -v 'http://localhost:12900/cluster/xxx/metrics/namespace/org.graylog2' -H 'Authorization: Basic VALID-CREDENTIALS-COPIED-FROM-GRAYLOG'`

returns:
```
< HTTP/1.1 400 Bad Request
< X-Graylog-Node-ID: xxx
< X-Runtime-Microseconds: 641170
< Content-Type: application/json
< Date: Fri, 05 Feb 2016 10:14:58 GMT
< Connection: close
< Content-Length: 220
< 
* Closing connection 0
{"type":"ApiError","message":"Unexpected character ('<' (code 60)): expected a valid value (number, String, array, object, 'true', 'false' or 'null')\n at [Source: java.io.InputStreamReader@332f027; line: 1, column: 2]"}
```

The problem seems to be caused by the rest_transport_uri, when I remove that it works fine. More specifically, it's caused by the custom path, if I remove that it works fine, even through an nginx proxy

**Comment by dennisoelkers** [comment]
Thanks for reporting this. Is there anything additional in your server logs?

**Comment by SjonHortensius** [comment]
Unfortunately not

**Comment by bernd** [comment]
It looks like this has been fixed in #1808. Please see our documentation on how to setup proxies.

http://docs.graylog.org/en/2.0/pages/configuration/web_interface.html

Thank you for the report!
